package-lock.json

```diff
@@ -1,13 +1,21 @@
 {
   "name": "system-administration",
+<<<<<<< Updated upstream
```

Comment:
Did you intend to commit this file?

Reply:
Not really, sorry about that - it is a mistake. Should I delete it or you can do it?

Reply:
I deleted this file.